src/Tooltip.jsx

```diff
@@ -61,8 +121,31 @@ class Tooltip extends Component {
     ]);
   }
 
-  getPopupDomNode() {
-    return this.trigger.getPopupDomNode();
+  adjustArrow = (align) => {
+    const { arrowStyle: arrowStyleCurrent } = this.state;
+    const targetNode = findDOMNode(this);
+
+    const arrowPlacement = pointNamesMap[align.points[0]];
+    const targetPlacement = pointNamesMap[align.points[1]];
+    const position = positions(
+      this.arrow,
+      arrowPlacement,
+      targetNode,
+      targetPlacement,
+    );
+
+    const arrowStyle = getArrowStyle(position, align);
+    if (
+      !arrowStyleCurrent ||
+      arrowStyle.top !== arrowStyleCurrent.top ||
+      arrowStyle.left !== arrowStyleCurrent.left
+    ) {
+      this.setState({ arrowStyle });
```

Comment:
Seems need keep setting this in case placement changed:
![Kapture 2019-05-17 at 10 51 53](https://user-images.githubusercontent.com/5378891/57900037-ea6b4380-7891-11e9-9e97-e6dade875db8.gif)